From 100f7ddafa0ca62efd5220985fd6f63c9e60d565 Mon Sep 17 00:00:00 2001
From: Uri Mashiach <uri.mashiach@compulab.co.il>
Date: Tue, 9 Jun 2020 10:12:17 +0300
Subject: [PATCH 59/60] arm: imx7d: cl-som-imx7: enable USB to Ethernet adapter
 AX88179

Enable the driver for the USB to Ethernet converter: USB3 ASIX AX88179

Signed-off-by: Uri Mashiach <uri.mashiach@compulab.co.il>
---
 include/configs/cl-som-imx7.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/configs/cl-som-imx7.h b/include/configs/cl-som-imx7.h
index 6fbee2186f..88c0166db7 100644
--- a/include/configs/cl-som-imx7.h
+++ b/include/configs/cl-som-imx7.h
@@ -216,6 +216,7 @@
 /* USB Networking options */
 #define CONFIG_USB_HOST_ETHER
 #define CONFIG_USB_ETHER_SMSC95XX
+#define CONFIG_USB_ETHER_ASIX88179
 #define CONFIG_USB_ETHER_RNDIS
 #define CONFIG_USB_ETHER_ASIX
 #define CONFIG_USB_ETHER_MCS7830
-- 
2.11.0

